pkgs/modules/python/pip.nix

```diff
@@ -1,7 +1,16 @@
 pkgs @ { pypkgs, ... }:
 
 let
-  pip = pypkgs.pip;
+  pip = pypkgs.pip.overridePythonAttrs (old: rec {
+    outputs = [
+      "out"
+    ];
+    # Skip building the docs for pip because that was failing
+    # with a sphinx-build error with a version of nixpkgs-unstable for 3.10 (worked for >3.11)
+    # and we don't need the docs
+    postBuild = "";
+    postInstall = "";
```

Comment:
Are we also going to lose [shell completions here](https://github.com/NixOS/nixpkgs/blob/9ddcaffecdf098822d944d4147dd8da30b4e6843/pkgs/development/python-modules/pip/default.nix#L96)? It looks like they aren't working anyway already though.

Reply:
Oops I can add them back in another pr (ci will be quicker).